src/Configuration/CFGCompiler/CFGCompiler.cpp

```diff
@@ -73,7 +74,7 @@ bool CFGCompiler::RegisterCommands(TclInterpreter* interp, bool batchMode) {
               cfgcompiler->m_cmdarg.tclOutput);
           cfgcompiler->m_cmdarg.tclOutput.clear();
         }
-        return result;
+        return result && cfgcompiler->m_cmdarg.tclStatus == TCL_OK;
```

Comment:
This condition doesn't work. Please use this:
```
if ((result != TCL_OK) || (cfgcompiler->m_cmdarg.tclStatus != TCL_OK))
  return TCL_ERROR;
return TCL_OK;
```

Reply:
I fixed it.